docs/architecture/architecture-fault-tolerance.rst

```diff
@@ -42,8 +42,7 @@ During a read operation, the coordinator communicates with just enough replicas
 
 .. image:: 5-read_op_RF_3_CL_2.jpg
 
-
-
+The diagram above illustrates the read operation where the coordinator V forwarded the read operation to X, Z and W, **respectively**, where Z and X failed to return its value, but W returned succesfully, ilustrated by the value **Read 1**, meeting the Consistency Level configured.
```

Comment:
first of all, thank you for contributing to Scylla. please take a look at https://github.com/scylladb/scylladb/blob/master/CONTRIBUTING.md#contributing-code-to-scylla , if you haven't done so.

s/succesfully/successfully/

also, i think, in general, we should use present tense in technical writing.

and, you might need to rework the commit message: use a more specific title, like "doc: explain read op diagram in more details". and squash these two commits into a single one. you might need to clone a local repo to do this.